test/session.ts

```diff
@@ -312,7 +330,7 @@ describe('Session', () => {
     it('should accept gaxOptions', done => {
       const gaxOptions = {};
       session.request = config => {
-        assert.strictEqual(config.gaxOpts, gaxOptions);
+        assert.deepStrictEqual(config.gaxOpts, {});
```

Comment:
```suggestion
        assert.strictEqual(config.gaxOpts, gaxOptions);
```

If after making that change, the test fails, then we would want to keep `deepStrictEqual`, but also set a property on `gaxOptions`.